2dfft_utils/misc/get_radius.py

```diff
@@ -0,0 +1,60 @@
+'''
+    get_radius.py
+    Purpose: Automates the process of finding the radii of simulation images. Uses numpy and astropy.
+    Inputs:  all_centers.txt (the output from get_center.py) and all FITS images in the working directory
+    Outputs: r_max.txt, a list of the galaxies' radii in the same order as all_centers.txt
+    Notes: Line 28 in the Radius function removes the scales superimposed on the image from the calculation.
+           You may need to modify the slice indices, depending on whether or not your image has scales and 
+           where they are located.
+           It is best to run this script from a directory containing a complete set of simulation images (0.000 - 3.000 Gyr).
```

Comment:
Let's make these instructions more general.  

Something like: While you can use `get_radius.py` in isolation, if you use it in conjunction with the other scripts here, you can't add/remove images in between steps (eg, you must use the same images & corresponding script output from beginning to end).